fcrepo-persistence-ocfl/src/test/java/org/fcrepo/persistence/ocfl/impl/FedoraToOCFLObjectIndexImplTest.java

```diff
@@ -48,4 +50,12 @@ public void test() throws Exception {
         assertEquals(ROOT_RESOURCE_ID, mapping1.getRootObjectIdentifier());
         assertEquals(OCFL_ID, mapping1.getOcflObjectId());
     }
+
+    @Test(expected = FedoraOCFLMappingNotFoundException.class)
+    public void testNotExists() throws Exception {
+        final FedoraToOCFLObjectIndexImpl index = new FedoraToOCFLObjectIndexImpl();
+
+
+        final FedoraOCFLMapping mapping1 = index.getMapping(RESOURCE_ID_1);
```

Comment:
you don't need to store the result to a variable here (causes warning for me)

Reply:
will remove.